pgjdbc/src/main/java/org/postgresql/replication/PGReplicationStream.java

```diff
@@ -17,7 +17,7 @@
  * It means that process wal record should be fast as possible, because during process wal record
  * lead to disconnect by timeout from server.
  */
-public interface PGReplicationStream {
+public interface PGReplicationStream extends AutoCloseable {
```

Comment:
Why don't you just use `Closeable`?

Reply:
`Closable` excepts an `IOException` to be thrown, not the super class `Exception`.